lib/elixir/lib/exception.ex

```diff
@@ -464,17 +464,15 @@ defmodule Exception do
       ""
 
   """
-  def format_file_line(file, line) do
-    format_file_line(file, line, "")
-  end
-
-  defp format_file_line(file, line, suffix) do
+  def format_file_line(file, line, suffix \\ "") do
```

Comment:
I like "(test)" suffix. This way we don't need to support `nil` here. Let's roll with that. :D